providers/implementations/kem/mlx_kem.c

```diff
@@ -103,6 +104,27 @@ mlx_kem_set_ctx_params(void *vctx, const OSSL_PARAM params[])
     return 1;
 }
 
+char *get_adjusted_propq(const char *propq)
```

Comment:
So we can end up with fips=yes,-fips?
This should have a clear comment that it just appends ",-fips" or returns -fips if not set.

Reply:
Added, thanks